docs/docsite/rst/intro_inventory.rst

```diff
@@ -179,6 +179,10 @@ Groups of Groups, and Group Variables
 It is also possible to make groups of groups using the ``:children`` suffix in INI or the ``children:`` entry in YAML.
 You can apply variables using ``:vars`` or ``vars:``:
 
+The order that the groups are in is important for the ini file format.
+You need to specify the child group then the parent group.
+The example below shows atlanta and raleigh then the southeast:children group they are in.
+If you specify the southeast:children then atlanta and raleigh it will fail parsing.
```

Comment:
this was a bug at one point, but should already be fixed

Reply:
When I opened this it was still an issue. That is why I made it a callout.

Reply:
understood, but then its a bug report, which others did, so we fixed it.

we do not normally change the docs to fit the bug

Reply:
I thought that it was intended. Once I discovered the issue it made sense that the individuals had to exist before you could group them.

Either way good to hear,